package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "zeplin-json-extension",
-    "version": "1.1.0",
+    "version": "1.1.1",
```

Comment:
No need to bump the version within the PR, will do that before releasing it!

Reply:
I’ll revert and leave the bumping up to you :)